README.md

```diff
@@ -105,7 +109,7 @@ find . -type f -execdir pywhat --include 'Bug Bounty' {} \;
 
 ```shell
 # Recursively download all web pages of a site
-wget -r -np -k https://skerritt.blog
+wget -U 'Opera 11.0' -r -np -k https://skerritt.blog
```

Comment:
Why the user-agent?

Reply:
as i said some websites require UserAgent for every request for security purpose such as requests using `curl` or `wget`. and when i tested on one of them, the wget command without -U didn't worked.